web/app/js/components/Sidebar.jsx

```diff
@@ -115,19 +130,45 @@ class Sidebar extends React.Component {
     }
   }
 
+  handleNamespaceSelector(value) {
+    this.setState({namespaceFilter: value});
+  }
+
+  // Filters resources retrieved from a stat query by namespaces. Does not include resources of type "authority"
+  filterResourcesByNamespace(resources, namespace) {
+    if (namespace === "all") {
+      return _.mapValues(resources, o => {
+        return _(o)
+          .filter(r => r.added)
+          .value();
+      });
+    }
+
+    return _.mapValues(resources, o => {
+      return _(o)
+        .filter(r => r.namespace === namespace && r.added)
+        .value();
+    });
+  }
+
+  generateResourceURL(r) {
```

Comment:
TIOLI: this is starting to pop up all over the place too... (here and in ResourceDetail) maybe we should put it in ApiHelpers....